src/components/input.js

```diff
@@ -289,9 +291,7 @@ export class AutocompleteTextInput extends Component {
         ])
       },
       renderSuggestion: v => v,
-      renderInputComponent: inputProps => {
-        return textInput({ ...props, ...inputProps, style, type: 'search' })
-      },
+      renderInputComponent: inputProps => h(TextInput, { ...props, ...inputProps, style, type: 'search' }),
```

Comment:
This should also use `nativeOnChange`, right now the workflow config is crashing.